docs/source/configuration.rst

```diff
@@ -75,25 +62,18 @@ Specify Configuration
 YAML files
 ~~~~~~~~~~
 
-You can specify configuration values in YAML files like the following:
+You can specify configuration values in YAML files. For example, if you want to modify 
+some of the workers memory configuration default values in distributed:
 
 .. code-block:: yaml
 
-   array:
-      chunk-size: 128 MiB
-
    distributed:
-     logging:
-       distributed: info
-       bokeh: critical
-       tornado: critical
-
-     scheduler:
-       work-stealing: True
-       allowed-failures: 5
+         workers: 
+            memory:
+               spill     : 0.85  # 0.7
+               target    : 0.75  # 0.6
+               terminate : 0.98  # 0.95
```

Comment:
This yaml looks right to me. I wonder if there is some way to make it clearer that the commented out values represent the defaults. Maybe:

```suggestion
               spill     : 0.85  # default: 0.7
               target    : 0.75  # default: 0.6
               terminate : 0.98  # default: 0.95
```

Reply:
That's a great suggestion : )